cylc/rose/rose.py

```diff
@@ -73,18 +78,51 @@ def get_rose_vars(dir_=None, opts=None):
     # Load the raw config tree
     config_tree = rose_config_tree_loader(dir_, opts)
 
+    # Extract templatevars from the configuration
+    get_rose_vars_from_config_node(
+        config,
+        config_tree.node,
+        os.environ
+    )
+
+    # Export environment vars
+    for key, val in config['env'].items():
+        os.environ[key] = val
+
+    return config
+
+
+# pre_configure entry point
+get_rose_vars = get_rose_vars_from_config_tree
```

Comment:
Au contraire.

Reply:
Missed that.
Happy now.

Reply:
Bit of a strange thing to do but I thought I'd save you a `pip install`.